framework/src/main/java/org/checkerframework/framework/util/ContractsUtils.java

```diff
@@ -418,16 +418,37 @@ private AnnotationMirror getAnnotationMirrorOfMetaAnnotation(
         for (Pair<AnnotationMirror, AnnotationMirror> r : declAnnotations) {
             AnnotationMirror anno = r.first;
             AnnotationMirror metaAnno = r.second;
-            List<String> expressions =
-                    AnnotationUtils.getElementValueArray(anno, "value", String.class, false);
-            AnnotationMirror postcondAnno = getAnnotationMirrorOfMetaAnnotation(metaAnno, anno);
-            if (postcondAnno == null) {
-                continue;
-            }
-            for (String expr : expressions) {
-                result.add(new Postcondition(expr, postcondAnno, anno));
+            try {
+                // If multiple contracts, then this will throw an exception which is catched at Line
+                // 433
+                List<String> expressions =
+                        AnnotationUtils.getElementValueArray(anno, "value", String.class, false);
+                AnnotationMirror postcondAnno = getAnnotationMirrorOfMetaAnnotation(metaAnno, anno);
```

Comment:
Please add tests to ensure that this code works correctly.  (In general, when a pull request changes code without adding a test, that is a code smell.)

The tests should include various cases, including the possibility that both the list and the singleton annotation are present, and that both the list and multiple singleton annoattions are present.